app/v2/notifications/post_notifications.py

```diff
@@ -131,11 +132,15 @@ def post_notification(notification_type):  # noqa: C901
     else:
         abort(404)
 
-    check_service_has_permission(notification_type, authenticated_service.permissions)
+    if not authenticated_service.has_permissions(notification_type):
+        raise BadRequestError(message="Service is not allowed to send {}".format(
```

Comment:
f-string instead?

Reply:
That code is copy-pasted from another file written before my time here, so I don't have a good answer for that.